app/src/main/java/com/ncnf/map/Marker.java

```diff
@@ -39,4 +44,6 @@ public String getTitle() {
     public String getSnippet() {
         return snippet;
     }
+
+    public List<Event> getEventList() {return eventList;}
```

Comment:
always return an unmodifiable list (using Collections library)

Reply:
Very true, thank you! I modified as you requested

Reply:
I agree